lib/honeycomb/client.rb

```diff
@@ -38,6 +38,7 @@ def initialize(configuration:)
         parser_hook: configuration.http_trace_parser_hook,
         propagation_hook: configuration.http_trace_propagation_hook,
       }
+      @error_backtrace_limit = configuration.error_backtrace_limit.to_i
```

Comment:
While the Configuration also does what it can to ensure this is an `Integer` at load/configuration time, it seemed good to just try and convert this here on the safe side.